field.go

```diff
@@ -368,7 +368,20 @@ func Stack(key string) Field {
 	// from expanding the zapcore.Field union struct to include a byte slice. Since
 	// taking a stacktrace is already so expensive (~10us), the extra allocation
 	// is okay.
-	return String(key, takeStacktrace())
+	return String(key, takeStacktrace(nil))
+}
+
+// StackSkip constructs a field that stores a stacktrace of the current
```

Comment:
Rather than duplicate the comments, I'd suggest just referring to `Stack`, `StackSkip is similar to Stack, but also skips the given number of frames from the top of the stacktrace" or something similar